packages/node/src/plugins/CommonJsChunkLoadingPlugin.ts

```diff
@@ -113,6 +118,38 @@ class CommonJsChunkLoadingPlugin {
             }
           );
 
+        compilation.hooks.runtimeRequirementInTree
+          .for(RuntimeGlobals.publicPath)
+          .tap("RuntimePlugin", (chunk, set) => {
```

Comment:
Tapping the same name as webpack internals will allow us to replace the existing codeGeneration output of webpacks internal PublicPathRuntimeModule

Reply:
This could also be achieved using `intercept` - which enables us to proxy internal hooks or other plugin hooks. But its called more times than just replacing the tap